ibis/expr/rewrites.py

```diff
@@ -182,7 +182,7 @@ def window_merge_frames(_, frame):
     group_by = tuple(toolz.unique(_.frame.group_by + frame.group_by))
 
     order_by = {}
-    for sort_key in _.frame.order_by + frame.order_by:
```

Comment:
Can you pill this change out to a separate PR?

Reply:
Doesn't seem worth it, I already fixed it upstream, this is just porting it to TES.